client/src/components/History/HistoryPublishedList.vue

```diff
@@ -0,0 +1,233 @@
+<script setup>
+import UtcDate from "components/UtcDate";
+import Tags from "components/Common/Tags";
+import { safePath } from "utils/redirect";
+import { computed, ref, watch } from "vue";
+import Heading from "components/Common/Heading";
+import LoadingSpan from "components/LoadingSpan";
+import DebouncedInput from "components/DebouncedInput";
+import { getPublishedHistories, updateTags } from "./services";
+import { getFilters, toAlias } from "store/historyStore/model/filtering";
```

Comment:
We should refactor this a little so other published components can use the `filterText` formatting without having to import history store internals. I think we should move parts of this functionality to a shared directory too, only the `validFilters` should probably remain, since that is history specific.